index.js

```diff
@@ -2,6 +2,7 @@ var fs = require('fs');
 var path = require('path');
 var join = path.join;
 var mkdirp = require('mkdirp');
+var pathIsAbsolute = require('path-is-absolute');
```

Comment:
I guess I was referring to this:
https://nodejs.org/api/path.html#path_path_isabsolute_path